### ISSUE TITLE
Partial Rendering

### ISSUE THREAD

**Issue by gliechtenstein** [Jasonette/JASONETTE-iOS#217]
Come up with a simple way to do partial rendering so that certain items on the screen can smoothly update without affecting rest of the view

**Comment by drwasho** [comment]
@hoffmabc

**Comment by hoffmabc** [comment]
Is this what it's come to @drwasho? lol

**Comment by drwasho** [comment]
Yes, I make the app, you help the platform lol @hoffmabc

**Comment by gtramontina** [comment]
I _have_ to leave a comment on ☝️ …

This is exactly the direction I, personally, would like to see mobile development going. Countless teams spend years and a sh*t load of money building rigid applications that can't really evolve. If, instead, teams put their efforts in building and improving such platforms we (community) don't have to reinvent the wheel every single time.

There are developers out there that would simply start bashing Jasonette for not supporting x or y and would ditch it for something "shinier" that would give them the feeling of "progress". Instead, you sit down and help to improve the platform. So props to you @drwasho and @hoffmabc for working this way. 👏 I was really looking forward to seeing this collaboration!

I can't thank @gliechtenstein enough for open sourcing Jasonette. Despite not having developed anything on with yet, or written any code to improve the platform, I truly believe this is a game changer. This is the level of decoupling that the web was built on and is all about. ❤️

**Comment by gliechtenstein** [comment]
Hahaha i had totally missed this priceless thread, just saw it 😂

Thanks @gtramontina I actually appreciate your support the most :)

Yes I think we all know something most others don't know, yet. I could be out promoting Jasonette all the time at conferences and stuff, but like you said, that will only make it feel "shinier" but not actually so. I consider this stage of Jasonette a "building era", and currently 100% focused on building. Jasonette itself hasn't even been out for a year so I think we're still early :)

At some point we'll go all out on promotion mode but for now I like the way it works because there's minimal distraction steering me away from actually building the core. I decided to work this way after having gone through a couple of times when Jasonette went viral, I couldn't get any work done during those times because I had to deal with all the newcomers.

Just to be clear there are all kinds of awesome people making use of Jasonette to build their apps, I know this guy who has hundreds of thousands of users who built his app using Jasonette on both iOS and Android, and there are many other cool use cases. I should probably write about these more, just didn't have time because writing takes a lot of time and effort, and my current priority is building.

TLDR: being shiny is not the focus of Jasonette at the moment, but all of us here know there's a diamond inside and take advantage of it, which is all that matters for now :)
